mautic/mautic
# Clearbit plugin

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Related user documentation PR URL | [141](https://github.com/mautic/documentation/pull/141)
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | /
| BC breaks? | N/A
| Deprecations? | N/A

This PR has the following PRs part of it:
- [x] #2905 
- [x] #2971

#### Description:
Plugin to pull data from Clearbit via the API about contacts and companies into Mautic.

#### Steps to test this PR:
1. Get a [Clearbit API key](https://dashboard.clearbit.com/signup)
1. ~~Apply this PR and the [Company SocialCache PR](https://github.com/mautic/mautic/pull/2971)~~ (already contains it)
2. Enable the plugin
![image](https://cloud.githubusercontent.com/assets/2924026/20487375/ad68a34a-afc8-11e6-8c49-776579476817.png)

3. On the contacts and on the companies pages, there are two buttons: One on the row dropdown menu, and another one on the toolbar at the top:
![image](https://cloud.githubusercontent.com/assets/2924026/20488164/b0337e3a-afcb-11e6-8994-c213c9852632.png)

4. When clicking one of those buttons a window will pop up to confirm the action:
![image](https://cloud.githubusercontent.com/assets/2924026/20521597/8f7e8ec2-b071-11e6-99c2-590cb90c227f.png)

5. A few moments later, if information was found, the contact/company details will be updated.
